common/compositor/vk/vkshim.h

```diff
@@ -16,6 +16,7 @@
 #define VKSHIM_H_
 
 #include <drm/drm_fourcc.h>
+#include <gbm.h>
```

Comment:
@gaochun Just had a chat about this with Kalyan, it seems like the most appropriate thing to do for now would be to rename this Vulkan format conversion function to something more generic (maybe NativeToVkFormat), and allow each platform to define their own version. So there would be version in os/linux/platformdefines.h that would do this GBM->VK conversion and a version in os/android/platfomrdefines.h that would do the appropriate thing for Android.